tasks/bower-concat.js

```diff
@@ -95,7 +96,7 @@ module.exports = function(grunt) {
 				var cssFiles = {};
 
 				_.each(lists.components, function(component, name) {
-					if (includes.length && _.indexOf(includes, name) === -1) return;
+					if (!options.includeWithDependencies && includes.length && _.indexOf(includes, name) === -1) return;
```

Comment:
Old behavior:

Inc.length | indexof==-1 | result
---------- | ----------- | ------
false | false | fall-through
true | false | fall-through
false | true | fall-through
true | true | return

New behavior

!iwd | Inc.length | indexof==-1 | result
---- | ---------- | ----------- | ------
false | same | as | above
true | will | always | fallthrough

In other words, if `!options.includeWithDependencies` is falsy (iwd=true) then it's as if `includes.length` is falsy in the old behavior.